geode-core/src/main/java/org/apache/geode/distributed/internal/direct/DirectChannel.java

```diff
@@ -493,7 +484,7 @@ private ConnectExceptions readAcks(List sentCons, long startTime, long ackTimeou
       }
       try {
         try {
-          con.readAck(msToWait, msInterval, processor);
+          con.readAck(processor);
```

Comment:
The timeout variables should be removed where they are declared above, since they're no longer used.